### PR TITLE
Add basic grade student table

### PR DESCRIPTION
**Pull request by nguyenGALAXY**

### CODE REVIEW

File: src/pages/DetailGrade/index.js [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+    }`,
+  },
+  '& .MuiDataGrid-cell': {
+    color:
+      theme.palette.mode === 'light'
+        ? 'rgba(0,0,0,.85)'
+        : 'rgba(255,255,255,0.65)',
+  },
+}))
+const theme = createTheme()
```
Comment (nguyen315) [inline src/pages/DetailGrade/index.js:80]: Theme is for app level, it use to make our source use the same config. Therefore, I think we should use only 1 theme & ThemeProvider at **app** level. Right now, I see there are 3 theme through our source code (Login.js, src/page/Profile.index.js and this file).  Could you refactor the 3 theme into main `index.js` file

File: src/pages/DetailGrade/index.js [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+        }
+        setListGradeStudent(listGradeStudent)
+        setUsers(response.data.filter((user) => user.role === 'STUDENT'))
+      } catch (error) {
+        console.log('Grade Detail Error', error)
+      }
+    }
+    getGradeDetail()
+  }, [])
+  console.log('User', rows)
```
Comment (nguyen315) [inline src/pages/DetailGrade/index.js]: remove console

File: src/pages/DetailGrade/index.js [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+  return (
+    <Layout>
+      <ThemeProvider theme={theme}>
```
Comment (nguyen315) [inline src/pages/DetailGrade/index.js]: could you refactor our source code to use 1 ThemeProvider

File: src/pages/DetailGrade/index.js [hunk 4]
```diff
@@ -0,0 +1,264 @@
...
+            alignItems: 'center',
+          }}
+        >
+          <CustomCard sx={{ minWidth: 1200 }} className={`sticky`}>
+            <CardHeader
+              avatar={
+                <IconButton onClick={goBack}>
+                  <ArrowBackIosNew />
+                </IconButton>
+              }
```
Comment (nguyen315) [inline src/pages/DetailGrade/index.js]: I get confuse at this code. Why CardHeader have avatar props and that avatar props is a back button 🤣

**Review by nguyen315 — COMMENTED** [review]
@nguyenGALAXY I have some comments

File: src/pages/DetailGrade/index.js (hunk 4 above)
Comment (nguyenGALAXY) [reply src/pages/DetailGrade/index.js]: Pls ignore it,  because according to CardHearder API, avatar props is a good spot for the goBack button.

**Pull request merged**